<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelaBarragem2D</class>
 <widget class="QDialog" name="janelaBarragem2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualização 2D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imagens/logo3.png</normaloff>../imagens/logo3.png</iconset>
  </property>
  <widget class="painelvisualizacaobarragem2dopengl" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>painelvisualizacaobarragem2dopengl</class>
   <extends>QWidget</extends>
   <header>painelvisualizacaobarragem2dopengl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
